repos/application-insights.proj

```diff
@@ -13,6 +13,9 @@
     <PropertyGroup>
       <BuildCommandArgs>$(ProjectDirectory)/Microsoft.ApplicationInsights.csproj</BuildCommandArgs>
       <BuildCommandArgs>$(BuildCommandArgs) /p:Configuration=$(Configuration)</BuildCommandArgs>
+      <!-- Re-assign EnlistmentRoot property so output directories end up under src/application-insights -->
+      <BuildCommandArgs>$(BuildCommandArgs) /p:EnlistmentRoot=$(ProjectDirectory)/src</BuildCommandArgs>
+      <BuildCommandArgs>$(BuildCommandArgs) /p:RelativeOutputPathBase=</BuildCommandArgs>
```

Comment:
I think Windows might need an explicit empty string here but I'm not sure.  It's an easy fix when we turn Windows on again if you don't care to experiment.  I'm fine merging as-is.